src/main/java/com/training/utilities/GetJsonValue.java

```diff
@@ -23,15 +22,8 @@ public String getValue(String key)  {
         } catch (IOException e) {
             e.printStackTrace();
         }
-
         // convert JSON file to map
         map = gson.fromJson(reader, Map.class);
```

Comment:
instead of declaring a variable gson, use this,
map = new Gson().fromJson(reader, Map.class);